### ISSUE TITLE
Dasboard scrollbar is too thin

### ISSUE THREAD

**Issue by vpanchal72** [grafana/grafana#17725]
Hi, 

After adding multiple panel to dashboards, Grafana adds own vertical scroll bar.
This customized bar is very thin, scroll bar width size should have to collected from client actice browser, or allow some where in config file to set width.

Let me know, if I am missing any setting 

Regards, 

Vijay Panchal

**What happened**:

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version:
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:

**Comment by dprokop** [comment]
@vpanchal72 thanks for reporting this. There is no option to configure scrollbars width in Grafana.

**Comment by erikvanoosten** [comment]
The issue states that the scroll bar should be configurable. I do not agree. I think Grafana should simply not try to change the scroll bar in any way. Leave this up to the browser (in tandem with the OS settings).

**Comment by ericmstone** [comment]
Either use OS defaults or make it configurable - this is a serious UI problem

**Comment by mmedvede** [comment]
I was trying to figure out why I always have problem scrolling in Grafana. It takes too much effort due to mangled side scrollbar. Is there any reason for Grafana to be implementing its own? I tried to set my browser options not to allow websites overriding scrollbar because of it, but it had no effect on Grafana UI.

**Comment by Jontron123** [comment]
I just started using Grafana recently and while trying to set up my first few panels, I noticed it was extremely cumbersome to navigate while setting them up. If everybody used a mouse with a scroll wheel it probably wouldn't be a big problem, but some of us are stuck using touch pads. The scrollbars are defiantly to small, which makes them hard to grab with the pointer. They also dont always like to appear when you need them. I find my self moving the pointer back and forth a lot trying to get them to appear. I agree with the others that the scrollbars should be the native OS/browser scroll bars. It would make navigation a whole lot easier.

**Comment by erikvanoosten** [comment]
Here is a workaround with a javascript bookmark.

Put the following in a bookmark:
```
javascript:(function(){var%20inputs=document.getElementsByClassName('css-52gpmd%20track-vertical');for(var%20i=0;i<inputs.length;i++){inputs[i].style.cssText='width:16px%20!important';}})();
```

Use the bookmark when you have Grafana open. It will overwrite make the vertical scroll bar 16 pixels wide.

Tested with Grafana 6.5.2 in Firefox for Mac.

**Comment by inflatador** [comment]
Any word on this one? It's tough to navigate with the skinny, disappearing scrollbars.

Thanks as always for your time!

**Comment by bdols** [comment]
sometimes the scroll wheel on my mouse isn't working (because I have a KVM switch that doesn't detect the scrollwheel movements sometimes)
when I try to use any of the scrollbars, it takes 10 seconds to get my mouse to select the scroll bar because it is a 2-3 pixel wide area that is able to be grabbed.
my experience shown in this video:

https://user-images.githubusercontent.com/286836/184174297-161f17a5-1536-4276-8e89-da31cc365e7c.MOV

**Comment by jmf-tls** [comment]
> sometimes the scroll wheel on my mouse isn't working (because I have a KVM switch that doesn't detect the scrollwheel movements sometimes) when I try to use any of the scrollbars, it takes 10 seconds to get my mouse to select the scroll bar because it is a 2-3 pixel wide area that is able to be grabbed. my experience shown in this video:
> 

Much worse than that: when sharing screen with Zoom, you get a thin green border; if the browser window is maximized it's basically impossible to get to the scroll bar!

**Comment by ashharrison90** [comment]
closing this as a dupe of https://github.com/grafana/grafana/issues/80429, as josh said you can track progress there 👍
